doc/COMPILING/COMPILING-VS-VCPKG.md

```diff
@@ -37,6 +37,10 @@ cd vcpkg
 .\vcpkg --triplet x64-windows-static install sdl2 sdl2-image sdl2-mixer[dynamic-load,libflac,mpg123,libmodplug,libvorbis] sdl2-ttf gettext
 ```
 
+If any package fails during build, for example `mpg123`, try installing it first.
+```cmd
+.\vcpkg install mpg123
```

Comment:
You better define triplet explicitly, otherwise it would install default triplet:

```suggestion
.\vcpkg --triplet x64-windows-static install mpg123
```

Reply:
I tried. It didn't work for me. But worked like this
```cmd
.\vcpkg install mpg123
.\vcpkg --triplet x64-windows-static install sdl2 sdl2-image sdl2-mixer[dynamic-load,libflac,mpg123,libmodplug,libvorbis] sdl2-ttf gettext
```

Reply:
Try deleting the vcpkg folder and installing it according to doc.

Reply:
Steps from our vcpkg instruction should work without any additional changes (unless some major change is done in vcpkg). If something does not work - there is and issue either with vcpkg itself or specific port. These are better to be addressed in vcpkg repo (https://github.com/microsoft/vcpkg).

For example, indeed there is an issue with `mpg123` now - it cannot find `yasm` (could be a regression from https://github.com/microsoft/vcpkg/issues/10523 or a new issue). When you try to build `mpg123:x86-windows` it tells you that you need `yasm-tool:x86-windows`:

```
  Cross-targetting and x64 ports requiring yasm require the x86-windows
  yasm-tool to be available.  Please install yasm-tool:x86-windows first.
```

When you run `vcpkg install mpg123` it uses default triplet (which is `x86-windows`, unless you have changed it with `%VCPKG_DEFAULT_TRIPLET%` environmental variable), thus it installs `yasm-tool:x86-windows`:

```
The following packages will be built and installed:
    mpg123[core]:x86-windows
  * yasm-tool[core]:x86-windows
  * yasm-tool-helper[core]:x86-windows
```

Then, when you run `vcpkg --triplet x64-windows-static install mpg123` again, `yasm-tool:x86-windows` would be available and build succeeds.

Reply:
Thanks for the clarifications. I need to delete these changes in this PR about failing package installation?